include/flags.hpp

```diff
@@ -367,7 +367,7 @@
 #define M_WILL_POISON               13      // Poisoner
 #define M_UNDEAD                    14      // Undead
 #define M_CANT_BE_STOLEN_FROM       15      // Cannot be stolen from
-// free                             16
+#define M_NO_HARM_SPELL             16      // Immune to Harm spell
```

Comment:
there's some place the flag names are defined, you'll need to add it there as well